src/jit/compiler.h

```diff
@@ -4801,7 +4801,7 @@ class Compiler
     //                  Create a new temporary variable to hold the result of *ppTree,
     //                  and transform the graph accordingly.
     GenTree* fgInsertCommaFormTemp(GenTree** ppTree, CORINFO_CLASS_HANDLE structType = nullptr);
-    GenTree* fgMakeMultiUse(GenTree** ppTree);
+    GenTree* fgMakeMultiUse(GenTree** ppTree, CORINFO_CLASS_HANDLE structType = nullptr);
```

Comment:
It was necessary to change `Compiler::fgMakeMultiUse` to use `CORINFO_CLASS_HANDLE structType = nullptr`  default parameter as nodes which structures require  `structType`. Otherwise, my code was hitting assertion:

```
Assert failure(PID 18536 [0x00004868], Thread: 8836 [0x2284]): Assertion failed 'structType != nullptr' in 'IntelHardwareIntrinsicTest.Program:Main(ref):int' (IL size 1093)

    File: e:\src\ms\dotnet\coreclr\src\jit\morph.cpp Line: 2725
```